packages/js/jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   coverageReporters: ['text-summary', 'lcov'],
   moduleFileExtensions: ['ts', 'js'],
   setupFilesAfterEnv: ['./jest.setup.js', 'jest-matcher-specific-error'],
+  testEnvironment: 'jsdom',
```

Comment:
@IceHe shall we align the usage of setting environment? I saw file level declarations as well.

Reply:
If I remember correctly, the file level declarations was just for the node environment. So declaring the default test environment as 'jsdom', and use file level declarations for node environment as an exception makes sense to me.

Reply:
why do we need file level "node" declaration?

Reply:
actually since js core SDK is not meant for dom only. we could leave it as default ?

Reply:
Please take a look at [this discussion in PR#137](https://github.com/logto-io/js/pull/137/files#r791602491)
Looks like we have a node env specific error handling

Reply:
js core SDK can be run in node environment.

Reply:
After offline team discussion, we decided to run tests for both jsdom and node envs, by executing the `run test` command twice in parallel in CI. Thus remove this env setting from jest.config.js